app/environments/brassbirmingham/brassbirmingham/envs/classes/board.py

```diff
@@ -508,11 +509,10 @@ def buildTwoRailroads(
     """
 
     def sellBuilding(
-        self, building: Building, buildLocation: BuildLocation, player: Player
+        self, building: Building, player: Player
```

Comment:
Shouldn't this be a `MarketBuilding`, if we're saying you can only sell those?

Reply:
Yes